docs/source/development.rst

```diff
@@ -63,12 +63,29 @@ Creating a grammar
 `antlr <https://www.antlr.org>`_, a parser generator. ``antlr`` expects grammar files
 and produces parsing Python code.
 
+Additionally, code generated using ``speedy-antlr-tool`` package is used to parse SQL scripts
+in C++ for better performance. The parsed tree is then converted into the Python equivalent.
+
 If you want to adapt the grammar please have a look at ``pytsql/src/pytsql/grammar/tsql.g4``.
-All other files in ``pytsql/src/pytsql/grammar`` are generated by ``antlr``. You can
-generate these files by running
+All files in ``pytsql/src/pytsql/grammar/cpp_src/antlr4-cpp-runtime`` are taken directly from
+the `ANTLR repository <https://github.com/antlr/antlr4/tree/master/runtime/Cpp/runtime/src>`_
+and the rest of the files in ``pytsql/src/pytsql/grammar`` are generated by ``antlr`` or
+``speedy-antlr-tool``. You can generate these files by running the following commands after
+adapting the path to your respective ``antlr`` jar file
+
+::
+
+    java -jar /usr/local/lib/antlr-4.9.2-complete.jar -Dlanguage=Cpp -visitor -no-listener -o cpp_src tsql.g4
+    java -jar /usr/local/lib/antlr-4.9.2-complete.jar -Dlanguage=Python3 -no-visitor -no-listener -o . tsql.g4
+
+And then running the ``speedy-antlr-tool`` using Python
```

Comment:
From which directory should this be run?

Reply:
Updated the docs to include the answer to this